src/applications/vaos/services/appointment/transformers.js

```diff
@@ -467,6 +495,45 @@ export function transformConfirmedAppointments(appointments) {
   });
 }
 
+function transformPendingCCAppointment(appt) {
+  const isCC = isCommunityCare(appt);
+  // const start = getMomentConfirmedDate(appt).format();
+  const minutesDuration = getAppointmentDuration(appt);
+
+  return {
+    resourceType: 'Appointment',
+    id: `var${appt.id}`,
+    // Required
+    status: getStatus(appt, isCC),
+    requestedPeriod: getRequestedPeriods(appt),
+    type: {
+      coding: [
+        {
+          // Why not typeOfCareId?
```

Comment:
Yeah, it might not be a bad idea for all requests to have the code here be `typeOfCareId` and the display be `appointmentType`. What do you think @narin?

Reply:
Yes, I think that makes sense

Reply:
Done.